example.env

```diff
@@ -1,11 +1,12 @@
-BS_LANG=en
+BS_DB_PASSWORD=PleaseChangeMe
+BS_LANGUAGE=en
 BS_URL=http://localhost
 BS_USER=WikiSysop
-BS_PASSWORD=PleaseChangeMe
+BS_SYSOP_PASSWORD=PleaseChangeMe
 HTTP_PORT=80
 HTTPS_PORT=443
 IMAGE_NAME=bluespice/bluespice-free
-IMAGE_TAG=4.2.x
+IMAGE_TAG=4.2.2
```

Comment:
its not required, we can even keep it empty and use any tag or image name while building/running it. I just kept it so that user can know that what is the version if there are multiple versions installed on a system